demo/clean_up/management/commands/clean_stale_db.py

```diff
@@ -0,0 +1,91 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils.timezone import now
+from sqlalchemy import text
+from sqlalchemy.exc import OperationalError
+
+from db import engine
+from db.metadata import get_empty_metadata
+from mathesar.models.base import Database
+from mathesar.state.django import reflect_db_objects
+
+
+class Command(BaseCommand):
+    help = 'Cleans up the stale database created during live demo'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--force',
+            action='store_true',
+            help='Force delete a database even if it in use'
+        )
+        parser.add_argument(
+            '--max-days',
+            action='store',
+            type=int,
+            help='A database is considered for deletion if it has existed for more than --max-days',
```

Comment:
I think you need to add a default here to avoid ending up with `max_days` set to `None`. I'm getting this error:

```sh
$ docker exec -it mathesar_service python manage.py clean_stale_db
Traceback (most recent call last):
  File "/code/manage.py", line 22, in <module>
    main()
  File "/code/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/code/demo/clean_up/management/commands/clean_stale_db.py", line 32, in handle
    drop_all_stale_databases(*args, **options)
  File "/code/demo/clean_up/management/commands/clean_stale_db.py", line 43, in drop_all_stale_databases
    stale_databases = Database.objects.filter(created_at__lt=now() - timedelta(minutes=max_days))
TypeError: unsupported type for timedelta minutes component: NoneType
```